docs/projects/frontend.md

```diff
@@ -0,0 +1,20 @@
+# Code Review Frontend
+
+The frontend is a Single Page Application, built with [Vue.JS](https://vuejs.org), using [vuex](https://vuex.vuejs.org/) as a store to get reactive programming share across components.
```

Comment:
`a store to get reactive programming share across components` what do you mean here?

Reply:
I just meant that data is synced across components. I rephrased it